rogue-thi-app/components/RoomMap.jsx

```diff
@@ -235,7 +233,8 @@ export default function RoomMap ({ highlight, roomData }) {
           <strong>
             {entry.properties.Raum}
           </strong>
-          {`, ${getTranslatedRoomFunction(entry?.properties?.Funktion, i18n)}`}
+          {entry?.properties?.Funktion_de !== '' ? ', ' : ''}
+          {getTranslatedRoomFunction(entry?.properties?.Funktion_de, i18n)}
```

Comment:
Why don't you use `Funktion_en` provided by the geojson, but translate the german function names using i18n?